.yamato/all-universal_split.yml

```diff
@@ -29,6 +29,8 @@ PR_Universal_Split_trunk:
          rerun: on_new_revision
       -  path: .yamato/all-urpupdate.yml#PR_URPUpdate_trunk
          rerun: on_new_revision
+    triggers:
```

Comment:
Do we want to reenable the urp jobs as well? Doesn't the team need to fix the build times?

Reply:
i thought itd be okay since it was just one from each project, but i will double-check with tim